editor/src/utils/feature-switches.ts

```diff
@@ -42,6 +44,7 @@ let FeatureSwitches: { [feature in FeatureName]: boolean } = {
   'Project Thumbnail Generation': false,
   'Debug - Print UIDs': false,
   'Debug – Connections': false,
+  'Data Entries in the Navigator': false, // !IS_TEST_ENVIRONMENT,
```

Comment:
Did you want to keep the comment?

Reply:
yep, because this is how we will want to enable it. I want to save myself 30 minutes when I turn it `true` then 70 tests fail, then I realize it should have been !IS_TEST_ENVIRONMENT and then I change it and then I wait for the tests to pass